tests/scripts/all.sh

```diff
@@ -5700,6 +5700,15 @@ component_check_python_files () {
     tests/scripts/check-python-files.sh
 }
 
+support_check_python_files () {
+    python3 - <<EOF
+import sys
+if sys.version_info.major >= 3 and sys.version_info.minor >= 6:
```

Comment:
We still have Python 3.5 on our CI. Until we upgrade, we need our Python scripts to work with Python 3.5.